textfinding_indikator9.py

```diff
@@ -0,0 +1,87 @@
+# this code is for TEXT FINDING - INDIKATOR 9
+# input is txt file from preprocess dokumen lama, nama instansi & judul from dokumen lama & baru
+
+import re
+import preprocess_dokbaru as dokbaru
+import preprocess_doklama as doklama
+from utility import *
+
+
+def txtreader(filename, lv, keyword):
+    # func to search keyword in txt file
+
+    # open txt file
+    file = open(f'cleaned_{filename}.txt', 'r')
+
+    idx = 0
+    result = []
+
+    # read line by line from txt
+    for line in file:
+
+        if (lv == 1):
+            for key in keyword:
+                reg = f'(?:(audit)\s({key}))'
+                if (re.search(reg, line, re.IGNORECASE)):
+                    result.append([idx, line])
+
+        else:  # lv == 2
+            # cek dgn keyword 'audit'
+            for key in keyword:
+
+                # manfaatin regex untuk searching yg not only kata keyword
+                reg = f'(?:(audit)\s({key}))'
+                if (re.search(reg, line, re.IGNORECASE)):
+                    result.append([idx, line])
+                    # hapus key yg udh ditemuin supaya gausah di search lagi
+                    keyword.remove(key)
```

Comment:
Correct me if I'm wrong, but I think for this case we don't neccessarily need to remove the keyword?

But if it turns out that it's needed to remove the keyword, then please change the for loop initialization at line 30 to `for key in list(keyword)` to avoid possible bugs

Reply:
Thank you, turns out removal is not needed! Please review the changes if you've time